addons/sale_management/models/sale_order.py

```diff
@@ -10,9 +10,14 @@
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
+    def _default_sale_order_template_id(self):
```

Comment:
a default that depends on other defaults (company), shouldn't it be a default_get override instead?